crates/rolldown/tests/rolldown/function/module_types/asset/artifacts.snap

```diff
@@ -4,11 +4,6 @@ snapshot_kind: text
 ---
 # Assets
 
-## assets/foo-vpuLoj-k.txt
-
-```txt
-hello, world
-```
```

Comment:
the asset did not generate properly

Reply:
It's expected. The test infra doesn't support print binary asset. Though `foo.txt` is actually not binary. It gets wrongly printed due to unexpected inner convertion here.

<img width="480" alt="image" src="https://github.com/user-attachments/assets/7366bc0f-c26e-40c7-96ff-1ac7dec97896">

It converts the inner bytes to string, which makes it printable and it shouldn't. Because `.txt` is treated as `asset` and we should assume if asset is text or binary.

However, I improve the test infra in https://github.com/rolldown/rolldown/pull/2561, so that it could at least print out what assets are generated.